src/components/Composer/index.js

```diff
@@ -363,18 +363,18 @@ function Composer({
         }
 
         if (textInput.current) {
-            textInput.current.addEventListener('paste', handlePaste);
+            document.addEventListener('paste', handlePaste);
```

Comment:
I know this is original code before function component refactor.
But I think it's better to set `textInput` as target as long as it doesn't cause any issues.
@bondydaa what do you think?

Reply:
yeah I was wondering about using `document` as well. 

my gut says having it on the dom element instead of the document object is probably better, in the event we add other global paste type handlers in the future so it won't create a conflict in that event.

Reply:
asked in open source here https://expensify.slack.com/archives/C01GTK53T8Q/p1692292251654389

Reply:
Hmmm previously I fixed an error where multiple composers existed at the same time like editing multiple messages and that's why handlePaste haves some logic of visibility. I'm not sure if maybe there is some logic behind that but I would prefer not to mess with it in case it does. Although if we move it to textInput we coudla void that logic of focus.
cc @mountiny in case you know there is logic about this

Reply:
FYI Here is the issue of multiple composers https://github.com/Expensify/App/issues/23502

Reply:
If we use it directly on the element itself instead of `document` then the below code seems also not required.

https://github.com/Expensify/App/blob/b7fdbcbff8327010d7e62ee588644c078a7bcabc/src/components/Composer/index.js#L266-L275

Reply:
I think we can go with this solution (document listener) for now as deploy blocker and discussion may not end up very soon.

Reply:
👍 thanks for that extra context and yeah agree let's go with this for now, we can always fix it later if needed too. this stuff isn't being carved into stone 😂

Reply:
YEah probably that would've been a possible solution for https://github.com/Expensify/App/issues/22803 maybe @Santhosh-Sellavel  can give us a clue on why it was fixed in document instead of textInput

Reply:
We used document only earlier